### DISCUSSION TITLE
Why can’t fields and enums in schema.prisma contain characters with diacritics?

### DISCUSSION THREAD

**Discussion by stefanbajic94** [prisma/prisma#11153] [Q&A] [answered]
If I’m using Postgres for example and it supports these characters I think it’d be good to have an option to enable such names.

**Comment by janpio** [comment, marked as answer]
You can use such names, but you need to use `@map` to define the database name and choose a "sanitized" version of that for your application. All identifiers (model, field, enum names) need to follow this limitation right now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
